### ISSUE TITLE
Investigate why ckpt are 135 MB

### ISSUE THREAD

**Issue by iibrahimli** [iibrahimli/msc_thesis#5]
Probably related to the internal self.model, which duplicates parameters. Should refactor all into a lightning module, it's a subclass of nn.Module anyway

**Comment by iibrahimli** [comment]
No, seems to be due to `save_weights_only=False` flag in ModelCheckpoint by default which also saves optimizer and scheduler states
